tests/test_curves.py

```diff
@@ -17,6 +17,20 @@
 class TestRZCurve:
     """Tests for FourierRZCurve class."""
 
+    @pytest.mark.unit
+    def test_center(self):
+        """Test center of curve."""
+        c = FourierRZCurve(R_n=[-2, 10, 4], Z_n=[1, 3, 2])
```

Comment:
also add rpz to this test to increase coverage

Reply:
That won't increase coverage because everything is already computed in rpz basis now by default, so computing in xyz effectively covers both cases. In this test it was easier for me to specify the "correct" center in xyz coordinates. FYI patch coverage is already at 100%.

Reply:
didn't realize, I thought a few spots said it was not covered